### PR TITLE
New theme + rewrite of dev documentation

### PR DESCRIPTION
**Pull request by chrissvo**
Fixes: https://github.com/mgmco/bugs/issues/1935
Fixes: https://github.com/mgmco/bugs/issues/2030

Changes proposed in this pull request:
* TSIA

### CODE REVIEW

**Comment by harianus** [comment]
Didn't see this button?

File: _sass/_clients.scss [hunk 1]
```diff
@@ -0,0 +1,53 @@
+// _sass/_clients.scss
+
+/** MGM COLORCODING **/
+.info.mgmco-green-light {
+	border-color: #c6e4de;
```
Comment (harianus) [inline _sass/_clients.scss]: ### Tabs?

File: _sass/_clients.scss [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+.lightbox img {
+	/** Pad the lightbox image */
+	max-width: 90%;
+	max-height: 80%;
+	margin-top: 2%;
+	background-color: #ffffff;
+	border-radius: 10px;
+}
+.lightbox:target {
+	/** Remove default browser outline */
```
Comment (harianus) [inline _sass/_clients.scss]: Blablabla

File: _config.yml [hunk 3]
```diff
@@ -1,25 +1,47 @@
...
+    position: 4
+  campaign:
+    title: Campaign
+    position: 5
+gems:
+  - jekyll-sitemap
+  - jekyll-seo-tag
 
+exclude:
+  - readme.md
```
Comment (harianus) [inline _config.yml]: Does this include `README.md`?

File: readme.md [hunk 4]
```diff
@@ -0,0 +1,39 @@
+# Client Documentation
```
Comment (harianus) [inline readme.md]: There is also an `README.md`, very confusing mister.

File: search.html [hunk 5]
```diff
@@ -0,0 +1,39 @@
...
+
+<div class="search-page">
+	<h2>Search Results</h2>
+	
+	<p><span id="search-process">Loading</span> results <span id="search-query-container" style="display: none;">for "<strong id="search-query"></strong>"</span></p>
+	<ul id="search-results"></ul>
+</div>
+
+<script>
+	window.data = {
```
Comment (harianus) [inline search.html]: I like this serverless search

File: _sales_tracking/marketeers.md [hunk 6]
```diff
@@ -0,0 +1,70 @@
...
+__Developers__: This information is probably very useful, but if you are looking for the technical specs, checkout [Information for developers]({{ site.baseurl }}/#sales_tracking_developers) ;)
+{: .error }
+
+#### Why sales tracking?
+
+We will pay out rewards and communicate on your behalf with both the promoter and the buyer. 
+
+1. We need to identify the buyer
+2. We need a way to connect that particular buyer to the right promoter
+3. We need to communicate with the buyer to pay out her reward.
```
Comment (chrissvo) [inline _sales_tracking/marketeers.md]: I'm fighting 1000s of years of gender inequality here ;p

File: _sales_tracking/marketeers.md [hunk 7]
```diff
@@ -0,0 +1,70 @@
...
+1. A promoter of your product tips your offer to a friend
+2. The friend visits the promotion page, loves what she sees (your product is awesome) and clicks the “Buy” button
+3. She enthusiastically enters your online sales funnel
+4. She successfully purchases your product! Yay!
+5. She did not cancel her contract and proved creditworthy. It is time to pay the promoter and her friend their rewards!
+
+Seems easy, but there is a lot going on behind the scenes! Let's go through the same steps:
+
+1. We are tracking how many potential buyers are visiting
+2. On the interstitial we create a sale. After 5 seconds the buyer is redirected to your website and the `MGMco sale ID` is present in the `URL`.
```
Comment (chrissvo) [inline _sales_tracking/marketeers.md]: Don't understand what you mean

File: js/main.js [hunk 8]
```diff
@@ -0,0 +1,101 @@
+jQuery(function() {
+  var $sidebar = $('#sidebar'),
```
Comment (mtimofiiv) [inline js/main.js]: those `$` variables! must be an angular developer.

**Pull request merged**